recipe/meta.yaml

```diff
@@ -102,6 +99,7 @@ test:
 about:
     home: https://simtk.org/home/simbody
     license: Apache-2.0
+    license_file: LICENSE.txt
```

Comment:
There are new fields here that could be helpful for users: `dev_url` should point to the github simbody repo, `doc_url` can point to any online documentation simbody might have.

Reply:
Done.